nexus3/install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+NEXUS_NS=nexus3
+
+kubectl create namespace $NEXUS_NS
+kubectl config set-context --current --namespace=$NEXUS_NS 
+
+helm repo add sonatype https://sonatype.github.io/helm3-charts/
+helm repo update
+
+helm install -f https://raw.githubusercontent.com/kameshsampath/kubernetes-marketplace/master/nexus3/values.yaml --wait \
```

Comment:
https://raw.githubusercontent.com/civo/kubernetes-marketplace/master/nexus3/values.yaml
Please change to this